gapis/capture/decoder.go

```diff
@@ -184,3 +184,15 @@ func (d *decoder) decode(ctx context.Context, in proto.Message) (interface{}, er
 
 	return obj, nil
 }
+
+func (d *decoder) flush(ctx context.Context) {
+	for _, g := range d.groups {
+		if obj, ok := g.(*cmdGroup); ok {
```

Comment:
Maybe just call `d. EndGroup` so that you have less copy-paste code, and so it will also work with future group types?